src/dtos/http_request.go

```diff
@@ -0,0 +1,15 @@
+package dtos
+
+import "github.com/Real-Dev-Squad/wisee-backend/src/models"
+
+type CreateUpdateFormRequestDto struct {
+	Status        string             `json:"status"`
+	Content       models.FormContent `json:"content"`
+	PerformedById int64              `json:"performed_by_id"`
```

Comment:
- what is `PerformedById`? If this is the if of the person creating/editing a form why does the frontend have to send it, can we not get it from the auth token?

Reply:
using it currently, as we have not setup the authentication.
The moment that is done we can remove this, this is just for the testing phase.